core/iwasm/common/wasm_runtime_common.c

```diff
@@ -1357,24 +1357,44 @@ wasm_runtime_unload(WASMModuleCommon *module)
 #endif
 }
 
+uint32
+wasm_runtime_get_overwritten_max_mem(uint32 max_memory_pages,
+                                     uint32 module_init_page_count,
+                                     uint32 module_max_page_count)
+{
+    if (max_memory_pages < module_init_page_count) {
+        LOG_WARNING("Cannot override max memory with value lower than module "
```

Comment:
Not sure is it better to use `LOG_WARNING("warning: ...")`?

Reply:
If the user is overriding the logger (i.e. `WAMR_BH_LOG`), they probably already have something to distinguish between different log levels: different colors (red for error, yellow for warning) or different log suffixes (e.g. `W`, `E` suffixes). So adding the log level to the log message is redundant.

Reply:
OK, got it, thanks.